pkg/core/blockchain.go

```diff
@@ -876,10 +876,41 @@ func (bc *Blockchain) GetTransationResults(t *transaction.Transaction) []*transa
 	return results
 }
 
+// GetScriptHashesForVerifyingClaim returns all ScriptHashes of Claim transaction
+// which has a different implementation from generic GetScriptHashesForVerifying.
+func (bc *Blockchain) GetScriptHashesForVerifyingClaim(t *transaction.Transaction) ([]util.Uint160, error) {
+	hashes := make([]util.Uint160, 0)
+
+	claim := t.Data.(*transaction.ClaimTX)
+	clgroups := make(map[util.Uint256][]*transaction.Input)
+	for _, in := range claim.Claims {
+		clgroups[in.PrevHash] = append(clgroups[in.PrevHash], in)
+	}
+	for group, inputs := range clgroups {
+		reftx, _, err := bc.GetTransaction(group)
```

Comment:
nit: reftx -> refTx

Reply:
Ugh. This casing is still something that I can't naturally write, I'm getting used to doing it for function names, but for variables it's a bit harder.